### ISSUE TITLE
Showing date without time not working

### ISSUE THREAD

**Issue by sumnerboy12** [damongolding/immich-kiosk#114]
**Describe the bug**
If I pass `&show_date=true` in my URL I don't see the date. If I pass `&show_date=true&show_time=true` I see both date and time. Passing just `&show_time=true` shows the time only.

**To Reproduce**
1. Go to Kiosk URL with '&show_date=true'
2. No date shown

**Expected behavior**
Expect to see just the date rendered

**Your Kiosk version**
0.11.1

**Your Kiosk installation**
- Docker

**Your Kiosk ENV or config file**
```
KIOSK_IMMICH_API_KEY=secret
KIOSK_IMMICH_URL=http://immich_app:3001
```

**Any parms passed to the URL used to access Kiosk**
Detailed above

**Desktop:**
 - OS: Windows
 - Browser: Chrome
 - Version: 129.0.6668.90

**Comment by damongolding** [comment]
I will investigate tomorrow as this isn't expected behaviour.

**Comment by damongolding** [comment]
Fixed in 0.11.2
